GreenGarbageApp/app/src/main/res/layout/fragment_leaderboard.xml

```diff
@@ -64,6 +64,17 @@
             android:textColor="@color/white"
             android:text="Points"
             android:textSize="24sp"/>
+        <TextView
```

Comment:
Why is the review result on the scoreboard? What has the review with the game result to do?  Didn't we discussed to add the result of all reviews to the drawer?